pkg/config/schema/metadata.yaml

```diff
@@ -256,7 +256,7 @@ snapshots:
 resources:
   # Kubernetes specific configuration.
   - kind: "CustomResourceDefinition"
-    plural: "CustomResourceDefinitions"
+    plural: "customresourcedefinitions"
```

Comment:
why this change?

Reply:
The old way was wrong, but in a way that didn't matter because we didn't use it anywhere. Expected `plural` is all lower case, which is used by all the others.

For example:
```
$ kubectl get crd -oyaml | rg plural:
      plural: authorizationpolicies
      plural: backendconfigs
....
```